Documentation/design/installconfig.md

```diff
@@ -270,6 +268,7 @@ type InstallConfig struct {
 type Admin struct {
 	Email string `json:"email"`
 	Password string `json:"password"`
+    SSHKey string `json:"sshKey"`
```

Comment:
Indent is wonky here.  Maybe tabs vs. spaces?

Reply:
updated to spaces, seems like that's what other fields do.

Reply:
fixed other places too in the file.